processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -74,6 +77,76 @@
 {
   private volatile DateTime maxIngestedEventTime;
 
+  private static final Map<String, ValueType> TYPE_MAP = ImmutableMap.<String, ValueType>builder()
+      .put("Long[]", ValueType.LONG)
```

Comment:
@jon-wei I was just curious to know why it was there. No need to remove if it does not hurt anything.